massgen/subagent/manager.py

```
@@ -818,6 +821,254 @@ async def _execute_with_orchestrator(
+            async def stream_events():
+                if process.stdout:
+                    async for line in process.stdout:
+                        try:
+                            line_str = line.decode().strip()
+                            if line_str:
+                                event = MassGenEvent.from_json(line_str)
+                                await on_event(event)
+                        except json.JSONDecodeError:
+                            # Skip non-JSON lines (e.g., startup messages)
+                            pass
+                        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)